docs/configuration/conflict_resolution.rst

```diff
@@ -0,0 +1,35 @@
+.. include:: ../global.rst.inc
+
+
+Conflict Resolution Options
+===========================
+
+
+::
+
+   jobs:
+   - type: push
+     filesystems: ...
+     conflict_resolution:
+       initial_replication: most_recent | all | fail # default: most_recent
+
+     ...
+
+.. _conflict_resolution-initial_replication-option-send_all_snapshots:
+
+
+``initial_replication`` option
+------------------------------
+
+The ``initial_replication`` option determines which snapshots zrepl replicates if the filesystem has not been replicated before.
+If ``most_recent`` (the default), the initial replication will only transfer the most recent snapshot, while ignoring previous snapshots.
+If all snapshots should be replicated, specify ``all``.
```

Comment:
Am I correct in assuming this will still only handle snapshots matched by the job's regex? If so, we should probably mention it in this sentence.

Reply:
There is no such thing. Replication always sends all snapshots that it sees.